packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -53,6 +54,32 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
       console.warn(`[${componentName}] \`${main}\` and \`${sub}\` cannot be used together. \`${sub}\` will be ignored.`)
     }
   }
+  const resolveTrailingSlashBehavior = (
+    to: RouteLocationRaw,
+    resolve: (to: RouteLocationRaw) => RouteLocation & { href?: string }
+  ): RouteLocationRaw | RouteLocation => {
+    if (options.trailingSlash !== 'append' && options.trailingSlash !== 'remove') {
```

Comment:
would be worth early returning too if `to` is null or undefined

Reply:
`if (!to || ...)`?

Reply:
done